using-openspace/configuration/index.md

```diff
@@ -0,0 +1,51 @@
+# Configuration
+
+OpenSpace uses a configuration file which is called `openspace.cfg`. This file is located in the OpenSpace folder. This file determines:
+
+- The SGCT configuration (the type of projection and number of windows)
```

Comment:
How about removing the empty line after each point? makes no difference in the output 

Also, Mr @alexanderbock usually wants the list indented one step, so I guess that should be done for this file as well (?)